extensions/eclipse-che-theia-terminal/src/browser/terminal-widget/remote-terminal-widget.ts

```diff
@@ -73,6 +73,7 @@ export class RemoteTerminalWidget extends TerminalWidgetImpl {
   protected channel: OutputChannel;
   protected closeOutputConnectionDisposable: Disposable;
   protected processGone: boolean;
+  private terminalApiEndPoint: URI | undefined;
```

Comment:
I haven't found where you use this field
I see that it's passed as a parameter, like `this.createWebSocket(this.terminalApiEndPoint!)` 
so - can we remove it?

Reply:
Sorry, I see  - there are 2 different methods...